packages/compass-collection-stats/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  root: true,
+  extends: ['@mongodb-js/eslint-config-compass'],
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: ['./tsconfig-lint.json'],
+    sourceType: 'module',
+    allowImportExportEverywhere: true,
```

Comment:
Any reason we have these here? It would be nice if we could have as less derivations from the shared config as possible

Reply:
Would it be useful to re-generate the skeleton with the script and copy over sources and tests?

Reply:
We have store as js file. Didn't move it to typescript because it is reflux and maybe it is easier just get rid of it later.

Reply:
I will totally leave this up to Alena, I think it's already in a good shape and as far as I can see almost everything here is aligned except for those few small things that I noted (and they might be here for a reason, I just want to double-check). Starting from scratch at that point might be much more work

Reply:
> Didn't move it to typescript because it is reflux and maybe it is easier just get rid of it later.

I might just not have enough context so sorry if that was discussed already and there is a decision on that, but it feels a bit weird to me that we are leaving parts of the plugin in js if the plan of the project was to move the whole thing to typescript. If we wanted to enable writing new code in typescript, we didn't need to move everything else here to typescript, similar to what we did with compass-aggregations. Can ticket the work for moving this file to typescript and include it in the project if we are not doing this right now?